package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "playwright_repo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "test.js",
+  "type":"commonjs",
+  "scripts":  {
+    "test": "playwright test",
+    "build": "webpack",
```

Comment:
This was added as an example. Our project doesn't contain webpack or node server. So this line and the next one can be deleted

Reply:
"test": "npx playwright test" was added